resources/jupyter-markdown.css

```diff
@@ -0,0 +1,37 @@
+/*
+Found that the colors of the alert boxes do not change with different themes in jlab.
+Hence hardcoded them here.
```

Comment:
I think we should still change colors to match the theme. Otherwise the notebook error message boxes won't match the color of the error alerts.

Reply:
> I think we should still change colors to match the theme

Jupyter lab doesn't do this, we can add support for theming later. Lets work on that in a separate PR